go/src/koding/kites/kloud/main.go

```diff
@@ -55,6 +55,7 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
+	defer k.Close()
```

Comment:
It doesnt not do any flush or smth but i just wanted to close/free resources immediately. Thanks for the note. These come default in socialapi runner so didn't give a second thought about whether if kloud supports graceful shutdown or not.